File: Seminar5/HomeWork/Model/Rational/DivRational.java
package HomeWork.Model.Rational;

public class DivRational extends CalcRational {

    public DivRational() {

    }

    // do_it
    @Override
    public double result() {
        return x / y;
    }

    @Override
    public void setX(double value) {
        super.x = value;

    }

    @Override
    public void setY(double value) {
        super.y = value;

    }

}

File: FinalTaskCalculator/Model/Rational/DivRational.cs
namespace ModelRational
{

    public class DivRational : Rational
    {
        // do_it
        public override double Result()
        {
            return x / y;
        }

        public override void SetX(double value) => this.x = value;

        public override void SetY(double value) => this.y = value;
    }
}